.travis.yml

```diff
@@ -207,6 +209,20 @@ deploy:
     repo: camptocamp/c2cgeoportal
     branch: master
     condition: '"${TRAVIS_PULL_REQUEST}" = false'
+- provider: script
+  script: travis/deploy-docker
```

Comment:
To me, deploy means "start containers". Would be more clear to name that `publish-docker`...